server/src/database/queries/activeUser.js

```diff
@@ -0,0 +1,9 @@
+const connection = require('../config/connection');
+
+const activeUser = (id, admin) =>
+  connection.query(
+    'UPDATE bookinguser SET is_active = true, is_admin =$1 WHERE id=$2',
```

Comment:
We are gonna allow the admin to do each of the following operations;
activate, deactivate, make admin, remove admin status